### ISSUE TITLE
timeout with OPTIONs on NSS

### ISSUE THREAD

**Issue by bourgeoa** [solid-contrib/specification-tests#113]
https://github.com/solid-contrib/specification-tests/blob/6fe781855cdf0104909123ee0c28624f4bed4fd6/protocol/cors/preflight.feature#L27

https://github.com/solid-contrib/specification-tests/blob/6fe781855cdf0104909123ee0c28624f4bed4fd6/protocol/cors/preflight-requests.feature#L44

**Comment by edwardsph** [comment]
I'm not sure what you are asking here. If NSS times out on this request isn't that an NSS issue?

**Comment by bourgeoa** [comment]
@edwardsph 
I tried a lot of redirect options on NSS
- I did not find a solution on `express` to redirect http to https
  there is no issue to redirect container Url without slash to container Url with slash (and this is implemented in `NSS)`
- `solidcommunity.net` is using NGINX to redirect 301 from http to https for port 80 to port 443,
   but it does not work or I don't find the solution for `solidcommunity.net:8443`
   
 Is this functionality a MUST in solid or in http ?

**Comment by edwardsph** [comment]
https://solidproject.org/TR/protocol#server-tls-https-redirect
> When both http and https URI schemes are supported, the server MUST redirect all http URIs to their https counterparts using a response with a 301 status code and a Location header.

Would it perhaps be true to say that your instance does not support http - in which case the test does not apply?

**Comment by bourgeoa** [comment]
> Would it perhaps be true to say that your instance does not support http - in which case the test does not apply?

Yes. And in which case NSS now passes 100% of the conformance tests.
